Octokit.Tests/Clients/SearchClientTests.cs

```diff
@@ -1045,7 +1045,78 @@ public void TestingTheCreatedQualifier_Between()
                     Arg.Is<Uri>(u => u.ToString() == "search/issues"),
                     Arg.Is<Dictionary<string, string>>(d => d["q"] == "something+created:2014-01-01..2014-02-02"));
             }
+            [Fact]
```

Comment:
:lipstick: insert a newline between the `{` and the `[Fact]` to give each test some space